cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -932,23 +930,30 @@ <h2 class="h3 u-mb15">
             {% elif card.advance_apr_median is not none
                 and card.advance_apr_min != card.advance_apr_max %}
                 <p>
-                    You’ll only know your exact cash advance APR if you are
-                    approved for the card. The median cash advance APR for
-                    this card is {{ apr(card.advance_apr_median) }}.
+                    The median cash advance APR for this card is
+                    {{ apr(card.advance_apr_median) }}.
+                </p>
+            {% endif %}
+            {% if card.cash_advance_fee_for_each_transaction %}
+                <p>
+                    The cash advance fee is charged for each time you get a cash
+                    advance.
                 </p>
             {% endif %}
-            {# TODO: Explain this better #}
-            <p>
-               The cash advance fee is charged
-               {{ ' for each cash advance transaction' if
-                    card.cash_advance_fee_for_each_transaction else
-                    ' once per billing cycle'
-                }}.
-            </p>
         {% else %}
             <p>This card does not offer cash advances.</p>
         {% endif %}
     </div>
+    {% if card.balance_transfer_offered or card.cash_advance_apr_offered %}
+        <div class="o-card-details-section">
+            {{ speed_bump( {
```

Comment:
Just a heads up: https://github.com/cfpb/consumerfinance.gov/pull/8334 makes the speed bumps way bigger. I'm curious to see how it affects detail pages.

Reply:
@contolini, excellent timing, I just added a suggestion to #8334 to add a modifier to the speed bump macro that would let us use a more compact version on the details page (the version that looks amazing on the list page looks gigantic on the details page).